docs/upgrading.md

```diff
@@ -5,6 +5,16 @@ Below we describe the additional manual steps required to upgrade between differ
 
 # Open Autonomy
 
+## `v0.8.0` to `v0.9.0`
+
+Breaking changes
+
+- the `max_participants` have been removed. 
+- the `consensus_threshold` should now be specified in the `setup` parameters. If `null`, then it is calculated automatically.
+- The synchronized database is now `serializable` and `hashable`. This means that the data inserted into the database are now enforced to be primitive or non-primitive built-in types only, except for sets and `frozensets`. In essence, the data should be `json serializable`.
+- The usage of local tendermint chain is optional in the deployment setup, use `-ltm, --local-tm-setup` when building a deployment using `autonomy deploy build` command to include a local tendermint chain setup.
```

Comment:
I mentioned before. `local tendermint chain` is not great terminology. We call it Tendermint Consensus Gadget elsewhere cc @jmoreira-valory